README.md

```diff
@@ -20,8 +20,22 @@ Install like any other shard. Additionally you'll need gobject-introspection ins
 as well as the GObject based library you want to use and its typelib file needs to be
 available.
 
+### Debian
+
+```
+sudo apt install libgtk-3-dev libgirepository1.0-dev
```

Comment:
I don't really want to communicate Gtk as a dependency of this shard, the only real dependency is `libgirepository1.0-dev`, everything else is only needed as you use it. Maybe move that mention to the Sample section? :)